packages/renderer/src/lib/images/StartIcon.svelte

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" fill="#fff" viewBox="0 0 384 512"
```

Comment:
For fontAwesome icons we do have <Fa>

```ts
import Fa from 'svelte-fa/src/fa.svelte';
 <Fa icon="{faNameOfTheIcon}" />
 ```
 so I'm not sure we should duplicate svg there